doc/architecture/decisions/0002-store-subset-of-syndication-data.md

```diff
@@ -0,0 +1,28 @@
+# 2. Store subset of Syndication data
```

Comment:
Not sure if it's worth explicitly mentioning the transform as the only significant changes made by the ETL are structural or name changes.